packages/test/test-end-to-end-tests/src/test/noDeltaStream.spec.ts

```diff
@@ -156,6 +156,13 @@ describeFullCompat("No Delta stream loading mode testing", (getTestObjectProvide
 					url: containerUrl,
 				});
 
+				// Force the container into write mode to ensure a summary will be created
+				const dataObject = await requestFluidObject<ITestFluidObject>(
```

Comment:
This did not have an exception, but it was failing when setting the default connection mode to 'read'